templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{{ request.META.HTTP_REFERER }}" class="btn btn-outline-danger" role="button">Cancel</a>
```

Comment:
Using `request.META.HTTP_REFERER` for the 'Cancel' button's URL might not always be reliable, as it depends on the HTTP referrer header, which can be empty or manipulated. Consider using a more reliable URL, such as the driver detail page or driver list page, to ensure consistent behavior.